### DISCUSSION TITLE
Simple Dunno, Intranet Server ?

### DISCUSSION THREAD

**Discussion by inukaze** [mm201/pkmnFoundations#107] [Q&A]
Hi there, i just want to know if you don't have a version of this server for "GTS", "Battle Video" and if possible the Plaza. just for Intranet use, without any internet connection ?

Similar to the package on this website -> https://herc.ws/board/topic/16607-ragnarok-offline-newbie-pack-2021-make-your-ro-server-in-less-then-5-minutes/

i think too on the server for « Phantasy Star Blue Burst » 

on internet exist some packages semi-preconfigured for make intranet servers without internet connection. this server have some pre-packaged server for intranet use ?

**Comment by mm201** [comment]
Pre-configured, no. It's still too much of a WIP and I have other priorities. There's a [kind of an old tutorial](https://github.com/mm201/pkmn-classic-framework/wiki/Creating-a-pkmnFoundations-server) on setting up a private instance on the Wiki which should probably work. Note that you'll also need to run [altWFC](https://github.com/barronwaffles/dwc_network_server_emulator) and DNS, and proxy some of the altWFC services which run on high ports to instead serve to their corresponding honstnames. (full instructions on the altWFC wiki) 

If you want unmodded DSes to be able to connect, you'll need to follow the instructions to setup [ndsconstraint](https://github.com/KaeruTeam/nds-constraint). The required Wii certificate is still on the internet if you know where to look.

If you want WiFi Plaza to work, you'll need to run a peerchat server. A partial implementation exists [here](http://aluigi.altervista.org/papers.htm#peerchat) but the working version is a closed source Wiimmfi component.
